AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -123,6 +123,8 @@ ayedas.com.tr,toroslaredas.com.tr,baskentedas.com.tr##.footer_cerezpolitikasi
 enerjisa.com.tr##.enerjisa-cerez-politikasi-cookie-bar
 xoxowifi.com###cookies_message_container
 tylko.com##.ty-settings
+cn.momondo.com#$#div[id$="data-transfer-disclaimer-dialog"] { display: none !important; }
+cn.momondo.com#$#body.hideOverflow { position: static !important; overflow: auto !important; }
```

Comment:
It looks like they don't work here `https://www.cn.momondo.com/flights/seoul`.
Please check
```
cn.momondo.com#$#body { position: static !important; overflow: auto !important; }
cn.momondo.com#?#body > div:has(> div[aria-modal="true"] a[href="/privacy#cookies"])
```